infrastructure/modules/shared.bicep

```diff
@@ -0,0 +1,34 @@
+@allowed([ 'dev', 'prd' ])
+param environment string
+param location string = resourceGroup().location
+
+param organizationPrefix string
+param sharedResourcesAbbreviation string
+
+var isPrd = environment == 'prd'
+var settings = isPrd ? loadJsonContent('../prd.settings.json') : loadJsonContent('../dev.settings.json')
+
+resource appservicePlan 'Microsoft.Web/serverfarms@2022-03-01' = {
+  name: 'asp-${organizationPrefix}-${sharedResourcesAbbreviation}-${environment}'
+  location: location
+  sku: settings.appservicePlan.sku
+  kind: 'linux'
+  properties: {
+    perSiteScaling: true
+    reserved: true
+  }
+}
+
+module insightsModule 'insights.bicep' = {
```

Comment:
We are already deploying a log analytics workspace in the shared azure repository which all services can use for logs. I do not see a need that Shifty should have its own log analytics workspace - it shall use the shared one